build-chroot/bump-pinned-bazel.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e -u
+
+# This script bumps the version of bazel to a newer version.
+
+if [ "${1:-}" = "" ]
+then
+	echo "usage: bump-pinned-bazel.sh <version>" 1>&2
+	exit 1
+fi
+
+VERSION="$1"
+
+curl -L -o "bazel-${VERSION}.deb" "https://github.com/bazelbuild/bazel/releases/download/${VERSION}/bazel_${VERSION}-linux-x86_64.deb"
+curl -L -o "bazel-${VERSION}.deb.sig" "https://github.com/bazelbuild/bazel/releases/download/${VERSION}/bazel_${VERSION}-linux-x86_64.deb.sig"
+gpg --no-default-keyring --keyring ./bazel-keyring.gpg --trust-mode=always --verify "bazel-${VERSION}.deb.sig"
```

Comment:
script refers to files `bazel-keyring.gpg` and `install-bazel.sh` which only works when run from this directory.  Also might consider downloading temporary files to a tmpdir so that they don't clutter the working directory if verification fails.  See `debian-checksum.sh` for how I did this for debian